airbyte-webapp/src/views/Connector/ConnectorForm/components/Property/Control.tsx

```diff
@@ -33,6 +35,7 @@ export const Control: React.FC<ControlProps> = ({
   error,
 }) => {
   const [field, meta, helpers] = useField(name);
+  const canUseDatepicker = useFeature(FeatureItem.AllowUseDatepicker);
```

Comment:
This should not be a feature, but instad should just be an experiment using `useExperiment`. There is no reason why we'd want to have this environment dependant, what features are meant for, but all we want is an experiment toggle for this. `useExperiment` would be the right place for this.